p2p/synch/sync.go

```diff
@@ -7,6 +7,7 @@ package synch
 import (
 	"context"
 	"fmt"
+	"github.com/Qitmeer/qitmeer/core/blockdag"
```

Comment:
I am not quite understand why we need to add the `blockdag` package dependency if it's just an empty struct marker.
```
// This struct is empty
type Empty struct{}
```

Reply:
> I am not quite understand why we need to add the `blockdag` package dependency if it's just an empty struct marker.
> 
> ```
> // This struct is empty
> type Empty struct{}
> ```

https://github.com/Qitmeer/qitmeer/pull/559/commits/ccc17135aeb81f56305e4cf8ac4b993eb4c65b9f